bin/cli.js

```diff
@@ -483,6 +498,22 @@ program
     }
   });
 
+// Development mode
+program
+  .command("development-mode")
```

Comment:
What about:
```
silverfin development-mode -h <handle> // automatically run liquid test on file save (YAML or liquid)
silverfin development-mode --liquid // automatically update templates for any liquid files (reconciliation/shared part)
```
And we remove `--watch` flag from `run-test` ?
Also, any better naming instead of `development-mode` ?